aspnet-core/test/AbpCompanyName.AbpProjectName.Tests/UnitTests/FakeServices/FakeLogger.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using Castle.Core.Logging;
+
+namespace AbpCompanyName.AbpProjectName.Tests.UnitTests.FakeServices
+{
+    public class FakeLogger : NullLogger, ILogger
```

Comment:
Technically I guess the FakeLogger isn't really required for this PR, but I used it a lot for verifying logging in my unit tests.  I'm happy to remove it if you like.